R/aa_utils.R

```diff
@@ -12,11 +12,22 @@ NULL
 .drive <- new.env(parent = emptyenv())
 .drive$base_url_files_v3 <- "https://www.googleapis.com/drive/v3/files"
 .drive$base_url <- "https://www.googleapis.com"
+.drive$mimetypes <-
+  system.file("extdata", "mimetypes.csv", package = "googledrive") %>%
+  read.csv(stringsAsFactors = FALSE)
```

Comment:
yikes no I didn't mean to keep that - I accidentally merged it in with my last commit 🙈